endpoints.json

```diff
@@ -62,5 +62,16 @@
 				}
 			]
 		}
+	},
+	"POST /api/articles/:article_id/comments": {
+		"description": "serves an object of the newly posted comment from the comments table",
+		"exampleResponse": {
```

Comment:
don't forget to include the key you return the data on. You will find this helpful later down the line!